bin/insertMissingAlerts.ts

```diff
@@ -4,14 +4,30 @@ import { Alerts, User } from '../src/entity';
 
 (async () => {
   const con = await createOrGetConnection();
-  const usersQuery = con
+  const stream = await con
     .getRepository(User)
     .createQueryBuilder()
     .select('id')
     .where('id NOT IN (SELECT "userId" FROM public.alerts)')
-    .getQuery();
+    .stream();
 
-  await con
-    .getRepository(Alerts)
-    .query(`INSERT INTO public.alerts ("userId") ${usersQuery}`);
+  stream.on('data', async ({ id }: { id: string }) => {
+    console.log('inserting alerts for: ', id);
+    await con
+      .getRepository(Alerts)
+      .createQueryBuilder()
+      .insert()
+      .values({ userId: id })
+      .orIgnore()
+      .execute();
+    console.log('alerts created for: ', id);
+  });
+
+  await new Promise((resolve, reject) => {
+    stream.on('error', reject);
+    stream.on('end', () => {
+      console.log('stream finished');
+      resolve(null);
+    });
+  });
```

Comment:
just add missing `process.exit()` at the end

Reply:
Right. To not keep the terminal hanging. Let me add that quickly.